remotespark/kernels/kernelmagics.py

```diff
@@ -190,7 +195,7 @@ def cleanup(self, line, cell="", local_ns=None):
             self.ipython_display.send_error("When you clean up the endpoint, all sessions will be lost, including the "
```

Comment:
To use a string and doing whatever you want to do with it, it's better to save the string first in a new variable then use it.